net.rpdev.OpenTodoList.json

```diff
@@ -0,0 +1,60 @@
+{
+    "id": "net.rpdev.OpenTodoList",
+    "rename-icon": "OpenTodoList",
+    "rename-desktop-file": "OpenTodoList.desktop",
```

Comment:
Should be renamed upstream. Same for the icons.
https://standards.freedesktop.org/desktop-entry-spec/latest/
https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-id-generic